internal/e2e/node/module_resolution.spec.js

```diff
@@ -37,3 +39,19 @@ describe('node npm resolution', () => {
     expect(lib1some.a).toEqual('lib1 content');
   });
 });
+
+describe('node data resolution', () => {
+  it('should be able to resolve data files', () => {
+    const dataContent = fs.readFileSync(data_resolve_relative);
+    expect(JSON.parse(dataContent)).toEqual({ "value": 42 });
+  });
+  it('should throw when resolving missing data files', () => {
+    if (process.platform.startsWith('win')) {
+      // On Windows, file location is the original one and not sandboxed.
+      // This means we cannot correctly exclude relative paths that aren't part of the runfiles.
+      return;
+    }
+    // This file exists in the source folder but is not in the data array.
+    expect(() => require.resolve('./data/missing-data.json')).toThrow();
```

Comment:
this test fails with sandboxing disabled too right?
How about change the desc
`it('should throw when resolving files that are outside the sandbox')`?

Reply:
Yes, it should fail without sandboxing in general.